biztalk/core/azure-blob-adapter.md

```diff
@@ -0,0 +1,153 @@
+
+1.  In the BizTalk Server Administration console, right-click **Send Ports**, select **New**, and select **Static One-way send port**.
+
+    [Create a Send Port](../core/how-to-create-a-send-port2.md) provides some guidance.
+
+2. Enter a **Name**. In **Transport**, set the **Type** to **AzureBlobStorage**, and click **Configure** button. 
+
+3. Configure the **Azure Account** properties: 
+
+    |Use this|To do this|  
+    |---|---|  
+    | **Sign-in** | Sign into your Azure account |
+    | **Subscription** | Select the subscription that has your Azure storage account |
+    | **Resource Group** | Select your resource group that has your Azure storage account |
+
+4. Configure the **General** properties: 
+
+    |Use this|To do this|  
+    |---|---|  
+    | **Storage Authentication** | Select an authentication method. <ul><li>Typically, it's recommended to use a Shared Access Signature, which is also selected by default. You can input the Shared Access Signature connection string in the **Connection string** field to provide the authentication.</li> <li>If you are using Access keys as authentication method, a collection of storage account will be populated up in **Account** drop-down list. After you select the storage account, **Connection string** field will be automatically populated with your primary access key, which also known as **key1**. </li></ul><br />The following links are good resources to help you decide which authentication method is right for your scenario:<br/><br/>[About Azure storage accounts](https://docs.microsoft.com/azure/storage/common/storage-create-storage-account)<br/>[Using shared access signatures (SAS)](https://docs.microsoft.com/azure/storage/common/storage-dotnet-shared-access-signature-part-1) |
+    | **Blob container name** | Select the name of your Blob container from the drop-down list. The list should be auto-populated after the **Connection string** field is filled up. |
+    | **Blob name** | Name of the blob used by adapter to upload the message to Blob storage container. Macros can be used for Blob name similar to file adapter. For available Macros please refer to [Using macros in file names](restrictions-when-configuring-the-file-adapter.md#using-macros-in-file-names). |
+    | **Namespace for blob metadata** | Context properties of the message can be written to blob metadata if the namespace of the property matches this field. |
+
+    When finished, your properties look similar to the following: 
+
+    ![General properties](../core/media/BlobAdapter-send-General.png)
+
+5. Configure the **Advanced** properties:
+
+    |Use this|To do this|  
+    |---|---|  
+    |**Blob type**| Azure Blob adapter can upload message as different blob types as needed, visit [blob types](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction) for more information. |
+    |**Write mode**| In case of same blob name already exists, different **Write mode** are provided. |  
```

Comment:
1) Even if it is trivial, we must mention the different write modes
2) Possible to improve English articulation. How about - Use different **Write mode** settings to handle cases when given **Blob name** already exists

Reply:
No, I did not mean language improvements. 
What are the values of the Write Mode? And when should the reader use a specific value.